crates/eframe/src/web/events.rs

```diff
@@ -402,6 +402,23 @@ pub fn install_canvas_events(runner_container: &mut AppRunnerContainer) -> Resul
         &canvas,
         "wheel",
         |event: web_sys::WheelEvent, mut runner_lock| {
+            let mut push_raw_event = || {
+                let unit = match event.delta_mode() {
+                    web_sys::WheelEvent::DOM_DELTA_PIXEL => egui::MouseWheelUnit::Pixel,
```

Comment:
What is called a "pixel" on the web, we call "point" in egui

```suggestion
                    web_sys::WheelEvent::DOM_DELTA_PIXEL => egui::MouseWheelUnit::Point,
```